src/annotator/test/guest-test.js

```diff
@@ -1047,53 +1043,72 @@ describe('Guest', () => {
   });
 
   describe('#detach', () => {
+    let index = 0;
```

Comment:
This could use a slightly more descriptive name, eg. `nextTagId`.

Reply:
👍🏼